docs/running-calyx/fud2.md

```diff
@@ -17,13 +17,19 @@ You might then want to do something like ``ln -s `pwd`/target/debug/fud2 ~/.loca
 fud2 depends on [Ninja][].
 Install it using your OS package manager or by downloading a binary.
 
-Create a configuration file at `~/.config/fud2.toml`, using the path to your checkout of the Calyx git repository:
+### Configuration
 
+Run the following command to edit `fud2`'s configuration file (usually `~/.config/fud2.toml`):
+```
+fud edit-config
```

Comment:
Just FYI I think there's a small typo here, this should be `fud2`.

Reply:
Argh, right! I'm AFK so if you wamt, please open a PR